### ISSUE TITLE
debug log findings on install 

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#1092]
Experimenting for the auto-backup feature, I've encountered that after the "final" reboot (reboot @ end of first-boot.bash I believe), `/boot/openhabian.conf` is replaced with the default, and `/boot/first-boot.log` is gone or overwritten by the second boot run with new values from the default /boot/openhabian.conf which has e.g. debugmode=off so all the debug output I wanted to see is gone.

@ecdye any spontaneous idea? cc: @holgerfriedrich 

webserver.bash is missing,  build.bash copies it there, it's removed somewhere so this may be because I restarted unattended install by rm /opt/openHABian-*; reboot

```
2020-08-15_02:00:13_CEST [openHABian] Enable maximum debugging output
++ timestamp
++ date +%F_%T_%Z
+ echo -n '2020-08-15_02:00:13_CEST [openHABian] Starting webserver with installation log... '
2020-08-15_02:00:13_CEST [openHABian] Starting webserver with installation log... ++ command -v python3
+ [[ -x /usr/bin/python3 ]]
+ bash /boot/webserver.bash start
bash: /boot/webserver.bash: No such file or directory
+ sleep 5
++ ps -ef
++ pgrep python3
+ isWebRunning=
+ [[ -n '' ]]

```

**Comment by ecdye** [comment]
Hmm, no idea. That is odd, I have never observed that behavior, is it possibly as a result of some change you made?

Also, just to make my life simpler, could you base #1077 off of #1055, as I rewrote `backup.bash` in that PR. I already tested the changes I made to it there and it is working so I don't think I'll make any more major changes to it.

**Comment by mstormi** [comment]
How am I supposed to rebase on something that isn't in master yet?

Plus I'm heavily working on `backup.bash`. I'd rather suggest the opposite that you base your changes to build.bash on #1077

**Comment by mstormi** [comment]
ok, wegarding webif (now webserver) missing, that's correct as it deletes itself after execution.

I triggered re-install a couple of times by removing /opt/openHABian-install-successfull and reboot, so I saw that probably on the second or an even later boot run.
Meanwhile first-boot.log getting deleted did not happen again.
It was apparently due to the image I used. I've built a new one, now it does not happen any more.
